### PR TITLE
Fixes the dynamic content page

### PR DESCRIPTION
**Pull request by pablotransifex**
Problem and/or solution
-----------------------
Fix for the dynamic content page by merging locales + extra locales

How to test
-----------

Reviewer checklist
------------------

Code:
* [ ] Change is covered by unit-tests
* [ ] Code is well documented, well styled and is following [best practices](https://tem.transifex.com)
* [ ] Performance issues have been taken under consideration
* [ ] Errors and other edge-cases are handled properly

PR:
* [ ] Problem and/or solution are well-explained
* [ ] Commits have been squashed so that each one has a clear purpose
* [ ] Commits have a proper commit

### CODE REVIEW

File: src/javascripts/io.js [hunk 1]
```diff
@@ -79,6 +79,13 @@ module.exports = {
       return l;
     });
   },
+  appendLocales: locales => {
```
Comment (candrikos) [inline src/javascripts/io.js:82]: I think `extendLocales` makes more sense.

File: src/javascripts/zendesk-api/config.js [hunk 2]
```diff
@@ -60,6 +77,13 @@ var config = module.exports = {
       this.ajax('activatedLocales')
         .done(data => {
           this.activatedLocalesDone(data);
+          this.ajax('activatedExtraLocales')
+          .done(data => {
+            this.activatedExtraLocalesDone(data);
+          })
+          .fail(xhr => {
+            this.activatedLocalesFail(xhr);
+          });
```
Comment (candrikos) [inline src/javascripts/zendesk-api/config.js:86]: we probably need a comment here due to the nested chaining ...

**Review by candrikos — APPROVED** [review]
The code seems fine, plus we tested that together ...
I am not sure about proper product-wise behaviour.
Do we need to fetch those strange ghost langs or should we skip those? 
Let's move one as it is and we may revisit that one after unblocking the client.

File: src/javascripts/io.js (hunk 1 above)
Comment (pablotransifex) [reply src/javascripts/io.js]: done

File: src/javascripts/zendesk-api/config.js (hunk 2 above)
Comment (pablotransifex) [reply src/javascripts/zendesk-api/config.js:88]: done
